product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -6,15 +6,45 @@ The database parameters listed on the DB Configuration tab are also referred to
 
 The list of parameters is populated based on the type of database you selected on the **Operational Settings** tab when you created your cluster. For more information about the parameters for your database type:
 
--   For Postgres parameters, see [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
+- For PostgreSQL parameters, see [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
      
 
--   For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+- For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+
+-  For more information on the types of values parameters accept, see [Parameter Names and Values](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). The types apply to both PostgreSQL and EDB Postgres Advanced Server parameters.
+
 
 !!!note
 Not all database configuration parameters are supported by BigAnimal. Some parameters, such as `wal_level` and `restore_command`, are reserved for EDB to provide the managed database features of BigAnimal. 
 !!!
 
+## Using formulas for parameter values
+
+In addition to entering specific values for parameters, for some parameters you can specify formulas to automatically calculate a value. You can use formulas for parameters of type integer and real using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`. GUCs used in formulas must also be of type integer or real. 
```

Comment:
The wording around "integer or real" here confuses me a bit; many GUCs have types like seconds, milliseconds, or GBs, right? @edbmanniwood

Reply:
> using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`

`?` and `:` are not operators in the same sense as the other operators in the system - furthermore I'm not sure that you could ever use the boolean operators as the value of a formula, since boolean GUCs do not accept formula. I think we need more detail here, and probably more examples.

Reply:
Two other potential concerns:

1. All arithmetic is on 64-bit floats - that should probably be noted
2. Do we need to talk about order of operations?

Reply:
We probably also need to talk about the limits on recursion.

Reply:
Is this the correct way to address potential concern 1? "All arithmetic is done on 64-bit floating point values rounded to an integer result."

Reply:
@edbmanniwood I need your help with Ben's latest round of comments.

Reply:
Ben raises a good point about ? and :. Those are used in ternary expressions only. But they are useful; the sample equation uses a ternary expression! For operator precedence (a.k.a. order of operations) we are at teh mercy of https://github.com/PaesslerAG/gval, which at first glance doesn't seem to document operator precedence. Fortunately, it (and out library) support the all-important parenthesis, so the general advice would be: * and / are at the same operator precedence. + and - are at the same operator precedence. * and / are evaluated before + and - . _Use parenthesis for everything else._

Reply:
Right now we have a maximum recursion depth of 1000, though IMHO that should be smaller; someone (probably RyanO) just picked that number and I preserved that number through every iteration/enhancement of the GUC lexer/parser/evaluator, assuming one day we would decide what the correct number should be. We might just wan to mention that we have a guard on recursion depth but reserve the right to determine the exact depth of recursion. We could promise to allow "around 100".

Reply:
Made some changes in https://github.com/EnterpriseDB/docs/pull/2361/commits/9ee2970224513d0887e17ff217378045aac5ab68, but I don't think we are quite there yet. I left the mentions of integer and real, but added more detail around operators and recursion levels.